package.json

```diff
@@ -28,9 +29,19 @@
     "@types/chai": "^4.1.7",
     "@types/mocha": "^5.2.6",
     "chai": "^4.2.0",
+    "husky": "^1.3.1",
```

Comment:
You can update this to 2.x now, it drops support for node 6 but that's all (node 6 is EOL and as a dev dependency that's fine imo)